packages/dd-trace/test/appsec/iast/analyzers/template-injection-analyzer.pug.plugin.spec.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const { prepareTestServerForIast } = require('../utils')
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../../../../src/appsec/iast/iast-context')
+const { newTaintedString } = require('../../../../src/appsec/iast/taint-tracking/operations')
+
+describe('template-injection-analyzer with pug', () => {
```

Comment:
In addition to these already tested methods, we also detected vulnerability when using the `render` method (which internally calls `handleTemplateCache` which in turn calls `compile`).

I think it would be worth adding a test with `render` to check that it still works in future versions, since we are not implementing it implicitly.

Reply:
Done